### PR TITLE
add source: prismascans

### PR DESCRIPTION
**Pull request by julioreigen**
<h2> Changes </h2>

Add a new madara source: [Prisma Scans](https://prismascans.net/) (PT-BR)

<h2> Checklist </h2>

- [x] Updated source's version for individual source changes
- [x] Set appropriate `nsfw` value
- [x] Did not change `id` even if a source's name or language were changed
- [x] Tested the modifications by running it on the simulator or a test device 

<h2>Screenshots <h2>

| Source info | Browse | Filters |
|--|--| -- |
|  source info|browse  |filters  |

|  Search | Manhwa | Chapter |
|--|--| -- |
|  search tab|manhwa tab  |browse  |

### CODE REVIEW

File: src/rust/madara/sources/prismascans/res/filters.json [hunk 1]
```diff
@@ -0,0 +1,203 @@
...
+	{
+		"type": "title"
+	},
+	{
+		"type": "group",
+		"name": "Status",
+		"filters": [
+			{
+				"type": "check",
+				"name": "OnGoing"
```
Comment (Skittyblock) [inline src/rust/madara/sources/prismascans/res/filters.json:11]:
```suggestion
				"name": "Ongoing"
```

File: src/rust/madara/sources/prismascans/res/source.json [hunk 2]
```diff
@@ -0,0 +1,12 @@
...
+	"info": {
+		"id": "pt-br.prismascans",
+		"lang": "pt-br",
+		"name": "Prisma Scans",
+		"version": 1,
+		"url": "https://prismascans.net",
+		"nsfw": 1
+	},
+	"listings": [
+	]
```
Comment (Skittyblock) [inline src/rust/madara/sources/prismascans/res/source.json:11]:
```suggestion
	}
```

File: src/rust/madara/sources/prismascans/src/lib.rs [hunk 3]
```diff
@@ -0,0 +1,47 @@
...
+use madara_template::template;
+
+fn get_data() -> template::MadaraSiteData {
+	let data: template::MadaraSiteData = template::MadaraSiteData {
+		base_url: String::from("https://prismascans.net"),
+		description_selector: String::from("div.summary_content_wrap div p"),
+		alt_ajax: true,
+		..Default::default()
+	};
+	data
```
Comment (Skittyblock) [inline src/rust/madara/sources/prismascans/src/lib.rs:16]: should just be able to do this
```suggestion
	template::MadaraSiteData {
		base_url: String::from("https://prismascans.net"),
		description_selector: String::from("div.summary_content_wrap div p"),
		alt_ajax: true,
		..Default::default()
	}
```

File: src/rust/madara/sources/prismascans/res/Icon.png
Comment (Skittyblock) [inline src/rust/madara/sources/prismascans/res/Icon.png:1]: This should have a background, not transparent

File: src/rust/madara/sources/prismascans/res/source.json [hunk 4]
```diff
@@ -0,0 +1,12 @@
+{
+	"info": {
+		"id": "pt-br.prismascans",
+		"lang": "pt-br",
+		"name": "Prisma Scans",
+		"version": 1,
+		"url": "https://prismascans.net",
+		"nsfw": 1
```
Comment (Skittyblock) [inline src/rust/madara/sources/prismascans/res/source.json:8]:
```suggestion
		"nsfw": 2
```

**Review by Skittyblock — CHANGES_REQUESTED** [review]

**Comment by julioreigen** [comment]
The website is under maintenance at the moment, when it comes back I will see if the source will still be working

**Review by Skittyblock — APPROVED** [review]

**Pull request merged**